DependencyInjection/Compiler/AddProviderCompilerPass.php

```diff
@@ -33,13 +33,7 @@ public function process(ContainerBuilder $container)
         $this->attachProviders($container);
 
         foreach ($container->findTaggedServiceIds('sonata.media.provider') as $id => $attributes) {
-            $container->getDefinition($id)->addMethodCall('addFormat', array('admin', array(
-                'quality'       => 90,
-                'width'         => 200,
-                'format'        => 'jpg',
-                'height'        => false,
-                'constraint'    => true,
-            )));
+            $container->getDefinition($id)->addMethodCall('addFormat', array('admin', $container->getParameter('sonata.media.admin_format')));
```

Comment:
Please don't use long lines. If there is an horizontal scroll on github, please consider the line too long.